test/WebJobs.Extensions.CosmosDB.Tests/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -400,7 +402,7 @@ public static IEnumerable<ParameterInfo[]> GetParameters()
         // These will set ConnectionStringSetting, which LeaseConnectionStringSetting should also use by default
         private static class ValidCosmosDBTriggerBindigsWithChangeFeedOptions
         {
-            public static void Func1([CosmosDBTrigger("aDatabase", "aCollection", ConnectionStringSetting = "CosmosDBConnectionString", MaxItemsPerInvocation = 10, StartFromBeginning = true)] IReadOnlyList<Document> docs)
+            public static void Func1([CosmosDBTrigger("aDatabase", "aCollection", ConnectionStringSetting = "CosmosDBConnectionString", MaxItemsPerInvocation = 10, StartFromBeginning = true, StartFrom = "2020-11-25T22:36:29Z")] IReadOnlyList<Document> docs)
```

Comment:
Need to test scenarios where it does not parse correctly and if we add a validation to not be used with StartFromBeginning

Reply:
Added few combinations